<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PaintAreaWidget" name="paintArea" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1001</width>
      <height>701</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1034</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>&amp;Правка</string>
     </property>
     <addaction name="actionDelete"/>
     <addaction name="actionSelectAll"/>
    </widget>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>&amp;Вид</string>
     </property>
     <addaction name="actionChangeColor"/>
    </widget>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>&amp;Инструменты</string>
     </property>
     <addaction name="actionSelectTool"/>
     <addaction name="actionRectangleTool"/>
     <addaction name="actionCircleTool"/>
     <addaction name="actionTriangleTool"/>
     <addaction name="actionLineTool"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="menu_3"/>
    <addaction name="menu_4"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelectTool"/>
   <addaction name="actionRectangleTool"/>
   <addaction name="actionCircleTool"/>
   <addaction name="actionTriangleTool"/>
   <addaction name="actionLineTool"/>
   <addaction name="separator"/>
   <addaction name="actionChangeColor"/>
   <addaction name="actionDelete"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Выделить всё</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionChangeColor">
   <property name="text">
    <string>Цвет</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSelectTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Выделение</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionRectangleTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Прямоугольник</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionCircleTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Круг</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionTriangleTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Треугольник</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionLineTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Отрезок</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintAreaWidget</class>
   <extends>QWidget</extends>
   <header>paintareawidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
